src/main/drivers/flash.c

```diff
@@ -117,13 +117,14 @@ static bool flashQuadSpiInit(const flashConfig_t *flashConfig)
                 }
 #endif
             }
+
```

Comment:
This looks like it should be a bugfix in another PR.

Reply:
I thought about that, but it's specifically for the H750 as there are no other builds that use QUADSPI in BF so it's fine in this PR which is H750 specific.